### ISSUE TITLE
/tmp folder full of files and folders?

### ISSUE THREAD

**Issue by llavall** [RakipInitiative/ModelRepository#300]
Hi, I have realised that quite a number of files and folders is accumulating on the servers since the new FSKlab. Looks like the stuff written there is not removed after running a workflow. What is the supposed way of preventing the server from overflow?

I see for example:
R-readPropsTempFile-xyz.r
R-propsTempFile-xyz.Rout
jsnippet-xyz.jar
workingDirectory-xyz/

**Comment by schuelet** [comment]
At least some of the files we generate are not properly deleted since we were never sure, when the expected life of a resource should end (e.g. on saving the workflow or on closing KNIME?). For lokal users we could and should make sure, files are at least deleted on resetting the node that created them.
For the server, I propose we find a way to store all of our generated files/folders in the actual workflow folder using the knime:// protocol. But that requires changes in many nodes and could take a while.

**Comment by llavall** [comment]
Ja, on the server a copy of a workflow is created as a job and after Discard the job is removed and all files in it along with it. Therefore on the server it would be great to save everything inside of the job. Could be just a matter of a changed path?

Locally, the workflow is at it is and no copy created. That means all files stored inside the workflow will enlarge it accordingly. That could be wanted behavior for executed workflows and saved in that state. But it must be assured, that the data is overwritten and not accumulated each run.
